src/test/java/com/microsoft/sqlserver/jdbc/TestUtils.java

```diff
@@ -368,6 +368,11 @@ public static void dropTypeIfExists(String typeName, java.sql.Statement stmt) th
         dropObjectIfExists(typeName, "TT", stmt);
     }
 
+    public static void dropUserDefinedTypeIfExists(String typeName, Statement stmt) throws SQLException {
+        String sql = "IF EXISTS (select * from sys.types where name = '" + escapeSingleQuotes(typeName) + "') DROP TYPE " + typeName;
+        stmt.executeUpdate(sql);
```

Comment:
Can we just combine the lines here? Other functions in this file are combined. Just a nitpick tho, feel free to ignore.